app/src/main/java/com/ivy/wallet/RootActivity.kt

```diff
@@ -465,4 +469,78 @@ class RootActivity : AppCompatActivity(), RootScreen {
         val addTransactionWidget = ComponentName(this, widget)
         appWidgetManager.requestPinAppWidget(addTransactionWidget, null, null)
     }
+
+}
+//For pushing Notification to the user as a reminder at every 12AM
+// urging to fill in expenses, transactions and incomes
+fun createNotificationChannel(context: Context) {
```

Comment:
Extract this logic in a separate class and module that lives outside `:app`. For example, `:feature:reminders`